app/controllers/rapidfire/attempts_controller.rb

```diff
@@ -1,5 +1,6 @@
 module Rapidfire
   class AttemptsController < Rapidfire::ApplicationController
+    protect_from_forgery with: :null_session
```

Comment:
Not sure if I'll keep this here. Other options include subclassing the controller in our app, or passing in some config.